Libraries/Components/Keyboard/Keyboard.js

```diff
@@ -101,6 +101,10 @@ let Keyboard = {
    *
    * Note that if you set `android:windowSoftInputMode` to `adjustResize`  or `adjustNothing`,
    * only `keyboardDidShow` and `keyboardDidHide` events will available on Android.
+   * 
+   * Limitation for keyboardWillShow: On Android no event is fired when the keyboard is about
+   * to be opened. Instead you can use a layout listener to check if the height of the layout
```

Comment:
I think we can delete this new text and add clarification on to the end of the previous sentence, something like this but maybe better wording:

> only `keyboardDidShow` and `keyboardDidHide` events will available on Android because it is not possible to listen for `keyboardWillShow` or `keyboardWillHide` events in the underlying Android APIs.